### ISSUE TITLE
Shift clicking is buggy

### ISSUE THREAD

**Issue by Pilvinen** [haveric/RecipeManager2#56]
When you craft this recipe with shift-clicking you get the base item but you do not get the NBT tag from give command. I think it bypasses the whole command part.

When you click it without shift you get the NBT tag and everything is OK.

```
craft kartussi
paper + paper + paper
paper + air + paper
paper + paper + paper
= diamond_axe:23:1
@noresult
@command give {player} minecraft:diamond_axe 1 23 {HideFlags:31,display:{Name:"§fKartussi",Lore:["Kartussi on tyhjä."]},Unbreakable:1b}
```

**Comment by Pilvinen** [comment]
No, I was wrong. You actually you get both items.

It seems the noresult tag is not working. You get the item from the command and you get the item from the recipe.

**Comment by Pilvinen** [comment]
It seems for now you can get around this by setting 0 as amount. You will then get a ghost item with 0 amount + the item from the command. The ghost item will disappear when you try to do something with it.

Ie. "= diamond_axe:23:0"

**Comment by haveric** [comment]
This should be fixed in the v2.8
